examples/android-room/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.sentry.android.roomsample">
+
+    <application
+        android:allowBackup="false"
+        android:icon="@mipmap/ic_launcher"
```

Comment:
I believe you can remove the icon as well as all the assets.

Let's try to keep the app as smaller as possible 👍

Ideally you don't need any `res` file at all: https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/examples/android-gradle-kts/src/main/AndroidManifest.xml

Reply:
Just kept the layouts 👍